codec.go

```diff
@@ -566,3 +561,7 @@ func toDisfix(db DisambBytes, pb PrefixBytes) (df DisfixBytes) {
 	copy(df[3:7], pb[0:4])
 	return
 }
+
+func trimNullPrefixBytes(b []byte) []byte {
+	return bytes.TrimLeftFunc(b, func(r rune) bool { return r == 0x00 })
```

Comment:
It's weird to be using bytes.TrimLeft* here, as those are meant for UTF8 encoded strings.  I don't know the UTF8 spec, but not knowing it, there's no way to tell whether this has the same behavior as before.

Lets never use runes when bytes will do, since that introduces all kinds of complexity.  Please convert to a naive for-loop.